letterbomb.fcgi

```diff
@@ -1,4 +1,4 @@
-#!/usr/bin/python2
+#!/usr/bin/python3
```

Comment:
flup does not support Python3, so this whole file needs to be removed, it is useless. I would run this under gunicorn or uwsgi or something like that.

Reply:
I'll add a wsgi file.